omniscidb/ConfigBuilder/ConfigBuilder.cpp

```diff
@@ -574,6 +574,13 @@ bool ConfigBuilder::parseCommandLineArgs(int argc,
                              ->default_value(config_->debug.enable_automatic_ir_metadata)
                              ->implicit_value(true),
                          "Enable automatic IR metadata (debug builds only).");
+  opt_desc.add_options()("dump-after-all",
+                         po::value<short>(&config_->debug.dump_llvm_ir_after_each_pass)
+                             ->default_value(config_->debug.dump_llvm_ir_after_each_pass)
+                             ->implicit_value(0),
```

Comment:
I'd expect the implicit value to be a non-zero value because if you specify the option it means you want the dumps to be generated.